manifests/t/TUitTUWien/TUtoolbox/2021.0.0.332/TUitTUWien.TUtoolbox.yaml

```diff
@@ -0,0 +1,15 @@
+PackageIdentifier: "TUitTUWien.TUtoolbox"
+PackageVersion: "2021.0.0.332"
```

Comment:
```suggestion
PackageVersion: "TUtoolbox (332)"
```

Reply:
Can not change the version number. we will have a 2021.1.0.x this year and a 2022.0.0.x next year. It's the version of the installer exe and all parts will change.

Reply:
PackageVersion needs to match the one in Control Panel as Windows Package Manager uses this key for `winget list` and `winget upgrade`, otherwise, the PR won't be able to be approved.

Reply:
THX for pointing out the bug in the installer. New installer (333) available, new manifest uploaded for testing.